src/wallet/wallet2.cpp

```diff
@@ -5127,7 +5124,6 @@ std::string wallet2::exchange_multisig_keys(const epee::wipeable_string &passwor
     expanded_msgs.emplace_back(msg);
 
   // reconstruct multisig account
-  crypto::public_key dummy;
```

Comment:
This one is already fixed in one of the multisig PRs. Can you undo this change?

Reply:
yup